com.github.rhyme_player.rhyme.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 Abdallah-Moh <abdallah.mam29@gmail.com> -->
+<component type="desktop">
+    <id>com.github.rhyme_player.rhyme</id>
+    <metadata_license>CC0</metadata_license>
```

Comment:
the validator didn't complain, but we also want to have the `project_license` tag here, to indicate which license the application is under.